styles/modules.css

```diff
@@ -103,7 +140,7 @@ body {
   }
 
   .main-nav ul li {
```

Comment:
You make good use of the structure to avoid needing to give everything its own class/id!